.circleci/config.yml

```diff
@@ -171,20 +171,13 @@ jobs:
       RUST_LOG: 'debug'
     steps:
       - checkout
-      - run:
-          name: Install rustup and clang
-          # We are installing them at the same time because it is faster
-          # todo: Remove --ignore-checksums flag
-          command: choco install rustup.install llvm -y --ignore-checksums
-      - run:
-          name: Add target
-          command: rustup target add x86_64-pc-windows-msvc
-      - run:
-          name: Install target toolchain
-          command: rustup toolchain install stable-x86_64-pc-windows-msvc
       - run:
           name: Check Trin workspace
-          command: cargo check --workspace
+          # We are running all these together because this version of circleci windows has an environment variable bug
```

Comment:
Can you link to an explanation of the bug, or describe a bit more what's happening here if no reference is available? When we find this comment later, we might want to evaluate if the bug has been resolved and we can split the steps back out. We will appreciate if we leave ourselves more hints/direction on how to do that.

Reply:
resolved